README.md

```diff
@@ -169,5 +170,23 @@ OK
 +----------------+
 ```
 
+#### Data example upload
+[example/resources/data](example) directory contains data sample to upload. You can create tables via [beeline cli](https://cwiki.apache.org/confluence/display/Hive/HiveServer2+Clients#HiveServer2Clients-Beeline%E2%80%93CommandLineShell), how to use beeline-cli see section [verifying setup](#verifying-setup).
```

Comment:
```suggestion
The [example/resources/data](example/resources/data) directory contains a data sample you can upload. 
To upload data and create tables you can use the [beeline cli](https://cwiki.apache.org/confluence/display/Hive/HiveServer2+Clients#HiveServer2Clients-Beeline%E2%80%93CommandLineShell), see the sql queries bellow. However, if you're not familiar with the `beeline-cli`, see the [verifying setup](#verifying-setup) section.
```

Is this correct or is the data already uploaded at this point?

Reply:
The data files have been already uploaded at this point. All that user need to do is to create the table in hive, to verify that